src/NoteMouseHandler/NoteMouseHandler.js

```diff
@@ -7,16 +7,14 @@ function findMap(arr, func) {
 }
 
 export default class MouseHandler {
-  constructor(emitter, actionFactories, cursorHandler) {
-    this.emitter = emitter
+  constructor(actionFactories, cursorHandler, defaultCursor) {
     this.actionFactories = actionFactories
     this.cursorHandler = cursorHandler
+    this.defaultCursor = defaultCursor
   }
 
-  onMouseDown(e, ctx) {
-    this.action = findMap(this.actionFactories, factory =>
-      factory(e, ctx)
-    )
+  onMouseDown(local, ctx, e) {
+    this.action = findMap(this.actionFactories, f => f(local, ctx, e))
```

Comment:
__[jshint]__
*  'arrow function syntax (=>)' is only available in ES6 (use 'esversion

<!-- Comment by [SideCI](https://sideci.com) -->